docs/content/guides/author-apps/dapr/how-to-dapr-sidecar/snippets/app-sidecar.bicep

```diff
@@ -0,0 +1,26 @@
+import radius as radius
+
+@description('Specifies the environment for resources.')
+param environment string
+
+resource app 'Applications.Core/applications@2023-10-01-preview' = {
+  name: 'demo'
+  properties: {
+    environment: environment
+  }
+}
```

Comment:
This should be a command line `-a` argument instead of being defined in the file.